service/src/main/java/greencity/security/service/OwnSecurityServiceImpl.java

```diff
@@ -220,6 +220,10 @@ public SuccessSignUpDto signUpEmployee(EmployeeSignUpDto employeeSignUpDto, Stri
         employee.setShowShoppingList(true);
         List<String> positionNames = employeeSignUpDto.getPositions().stream()
             .map(PositionDto::getName).collect(Collectors.toList());
+        if (positionNames.isEmpty()) {
```

Comment:
Why don't we use both `name` and `naneEn` for `positionsNames`?

Reply:
changed method

Reply:
changed method